src/Vlingo.Common/Message/AsyncMessageQueue.cs

```diff
@@ -121,7 +118,34 @@ public virtual void Run()
             finally
             {
                 dispatching.Set(false);
+                endDispatchingEvent.Set();
+            }
+        }
+        
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+        
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposed)
+                return; 
+      
+            if (disposing) {
```

Comment:
Everywhere else the `{` starts from the next line. Would be nice to have it consistent as the rest of the code.

Reply:
Agreed